pkg/assets/assets_test.go

```diff
@@ -0,0 +1,62 @@
+// Copyright 2020 The Lokomotive Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// nolint:testpackage
+package assets
+
+import (
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"testing"
+)
+
+// Verify assets are successfully extracted from memory to disk.
+func TestExtract(t *testing.T) {
+	dir, err := ioutil.TempDir("", "lokoctl")
+	if err != nil {
+		t.Fatalf("Creating temp dir: %v", err)
```

Comment:
Test cases errors should be capitalized the same way as regular errors. I cannot find the reference to the guideline recommending that though :/
```suggestion
		t.Fatalf("creating temp dir: %v", err)
```